package.json

```diff
@@ -16,12 +16,20 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
     "@babel/runtime": "^7.0.0",
     "@sentry/browser": "^5.4.2",
     "@sentry/integrations": "^5.4.2",
     "@sentry/typescript": "^5.3.0",
     "@types/lodash": "^4.14.134",
-    "@types/react-dom": "^16.8.4",
+    "@types/moment-timezone": "^0.5.12",
```

Comment:
@HazAT Do you know if/why these need to be dependencies? Would be really nice to move these to devDependencies?

Reply:
Sorry for the late reply. I thought we need this since we do different stuff if you call 
`make develop` or `yarn install`.
`make develop` apparently on installs `dependencies` whereby `yarn install` installs all.

So if you do a prod build it would fail because of missing types.

Reply:
Gotcha, thanks for the explanation